### PR TITLE
Fix(Curriculum): Corrected tasks feedbacks and descriptions English Curriculum

### PR DESCRIPTION
**Pull request by nieldakarla**
Just fixed some issues reported by Anastasiia.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by DanielRosa74 — COMMENTED** [review]
There's also a `he` here. It needs to be changed to `she` as well.

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/657e07a2fdda8d5ac93d415e.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Listen to the audio to complete the sentence below.
 
 ### --feedback--
 
-Sophie is describing what he does next in his routine after the first activity he mentioned.
+Sophie is describing what he does next in her routine after the first activity she mentioned.
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-a-typical-workday-and-tasks/657e07a2fdda8d5ac93d415e.md:36]: There is also a `he` here. It needs to be changed to `she` as well.

**Review by DanielRosa74 — CHANGES_REQUESTED** [review]
Change suggested to line 36 of file 657e07a2fdda8d5ac93d415e.md

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md [hunk 2]
```diff
@@ -7,15 +7,15 @@ dashedName: task-25
 
 # --description--
 
-Before you learned to ask questions using `do,` you can also use the auxiliary verb to in negative sentences. In negative sentences you combine the auxiliary verb `do` (I, you, we, they) or `does` (he, she it) with the word `not` and place it right before the main verb:
+Before you learned to ask questions using `do,` you can also use it to create negative sentences by combining `do` (for I, you, we, they) or `does` (for he, she, it) with the word `not` and placing it right before the main verb:
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:10]: I'd suggest that the pronouns be surrounded by backslashes (e.g: `I`, `you`, `we`, `they`) in the parentheses.

**Review by DanielRosa74 — CHANGES_REQUESTED** [review]
Change suggested to 657b6f641e5c3ab1afc6efc1.md

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md (hunk 2 above)
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:10]:
```suggestion
Before you learned to ask questions using `do,` you can also use it to create negative sentences by combining `do` (for `I`, `you`, `we`, `they`) or `does` (for `he`, `she`, `it`) with the word `not` and placing it right before the main verb:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md [hunk 3]
```diff
@@ -7,15 +7,15 @@ dashedName: task-25
...
 
 `I do not work everyday` - (Subject + do + not + main verb)
 
 `It does not function` - (Subject + does + not + main verb)
```
Comment (nieldakarla) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:14]:
```suggestion

`It does not function` - (Subject + `does` + `not` + main verb)
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md (hunk 2 above)
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:10]:
```suggestion
Before you learned to ask questions using `do,` you can also use it to create negative sentences by combining `do` (for `I`, `you`, `we`, `they`) or `does` (for `he`, `she`, `it`) with the word `not` and placing it right before the main verb:
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md [hunk 4]
```diff
@@ -7,15 +7,15 @@ dashedName: task-25
 
 # --description--
 
-Before you learned to ask questions using `do,` you can also use the auxiliary verb to in negative sentences. In negative sentences you combine the auxiliary verb `do` (I, you, we, they) or `does` (he, she it) with the word `not` and place it right before the main verb:
+Before you learned to ask questions using `do,` you can also use it to create negative sentences by combining `do` (for I, you, we, they) or `does` (for he, she, it) with the word `not` and placing it right before the main verb:
 
 `I do not work everyday` - (Subject + do + not + main verb)
```
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:12]:
```suggestion
`I do not work everyday` - (Subject + `do` + `not` + main verb)
```

File: curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md (hunk 3 above)
Comment (DanielRosa74) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-conversation-starters-in-the-break-room/657b6f641e5c3ab1afc6efc1.md:14]:
```suggestion
`It does not function` - (Subject + `does` + `not` + main verb)
```

**Review by DanielRosa74 — APPROVED** [review]
LGTM

**Review by huyenltnguyen — APPROVED** [review]

**Pull request merged**